server/db/db.js

```diff
@@ -0,0 +1,7 @@
+const Sequelize = require('sequelize');
+const db = new Sequelize(
+  process.env.DATABASE_URL || 'postgres://localhost:5432/escapehatch', {
```

Comment:
We don't like hard-coded URLs unless they are aggregated in some kind of `config.js` or something, so it becomes very easy to find and change them.